packages/js/product-editor/src/components/variations-table/variation-actions-menus/quick-update-menu.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { Button, Dropdown } from '@wordpress/components';
+import { createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { chevronDown, chevronUp } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { VariationActionsMenuProps } from './types';
+import { VariationActions } from './variation-actions';
+
+export function QuickUpdateMenu( {
```

Comment:
I think we should refer to this as the `MultipleUpdateMenu`.

Reply:
Done in commit [71779a5](https://github.com/woocommerce/woocommerce/pull/43441/commits/71779a54331ed63185bc56f3c096ade1040f0363)